drivers/pm_cpu_ops/pm_cpu_ops_psci.c

```diff
@@ -66,19 +70,12 @@ int pm_cpu_on(unsigned long cpuid,
 	return psci_to_dev_err(ret);
 }
 
-int pm_system_off(void)
+#ifdef CONFIG_SHUTDOWN
+void z_sys_shutdown(void)
 {
-	int ret;
-
-	if (psci_data.conduit == SMCCC_CONDUIT_NONE) {
-		return -EINVAL;
-	}
-
-	/* A compliant PSCI implementation will never return from this call */
-	ret = psci_data.invoke_psci_fn(PSCI_0_2_FN_SYSTEM_OFF, 0, 0, 0);
-
-	return psci_to_dev_err(ret);
+	(void)psci_data.invoke_psci_fn(PSCI_0_2_FN_SYSTEM_OFF, 0, 0, 0);
```

Comment:
Here, we should check anyway for the conduit to be set.

Reply:
yes, that's fine as well.

Reply:
done